static/app/views/issueDetails/eventNavigation.tsx

```diff
@@ -0,0 +1,233 @@
+import styled from '@emotion/styled';
+import omit from 'lodash/omit';
+
+import {Button, LinkButton} from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import {TabList, Tabs} from 'sentry/components/tabs';
+import TimeSince from 'sentry/components/timeSince';
+import {IconChevron} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Event} from 'sentry/types/event';
+import type {Group} from 'sentry/types/group';
+import {defined} from 'sentry/utils';
+import {getShortEventId} from 'sentry/utils/events';
+import {getReplayIdFromEvent} from 'sentry/utils/replays/getReplayIdFromEvent';
+import {useLocation} from 'sentry/utils/useLocation';
+import {useNavigate} from 'sentry/utils/useNavigate';
+import useOrganization from 'sentry/utils/useOrganization';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+
+type EventNavigationProps = {
+  event: Event;
+  group: Group;
+};
+
+enum EventNavOptions {
+  RECOMMENDED = 'recommended',
+  LATEST = 'latest',
+  OLDEST = 'oldest',
+}
+
+const EventNavLabels = {
+  [EventNavOptions.RECOMMENDED]: t('Recommended Event'),
+  [EventNavOptions.LATEST]: t('Last Event'),
+  [EventNavOptions.OLDEST]: t('First Event'),
+};
+
+const eventDataSections = [
```

Comment:
nit: it's usually good to type this at the source (e.g. `const eventDataSections: SectionDefinition[]`), makes it easier on the TS compiler and also will give better error messages if someone modifies one of the properties.